sample-app/build.gradle

```diff
@@ -19,6 +19,26 @@ android {
         sourceCompatibility rootProject.ext.sourceCompatibility
         targetCompatibility rootProject.ext.sourceCompatibility
     }
+
+    buildTypes {
+        release {
+            minifyEnabled true
+        }
+    }
+
+    signingConfigs {
+        release {
+            storeFile file("android.jks")
+            storePassword "android"
+            keyAlias "RxAndroidSample"
+            keyPassword "android"
+        }
+    }
+    buildTypes {
```

Comment:
Two blocks for this? How about moving `signingConfigs` up and combining these two `buildTypes` blocks.